core/agents/deezer/deezer.go

```diff
@@ -0,0 +1,93 @@
+package deezer
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"strings"
+
+	"github.com/navidrome/navidrome/conf"
+	"github.com/navidrome/navidrome/consts"
+	"github.com/navidrome/navidrome/core/agents"
+	"github.com/navidrome/navidrome/log"
+	"github.com/navidrome/navidrome/model"
+	"github.com/navidrome/navidrome/utils/cache"
+)
+
+const deezerAgentName = "deezer"
+const DEEZER_API_PICTURE_XL_SIZE = 1000
+const DEEZER_API_PICTURE_BIG_SIZE = 500
+const DEEZER_API_PICTURE_MEDIUM_SIZE = 250
+const DEEZER_API_PICTURE_SMALL_SIZE = 56
```

Comment:
We don't use ALL_CAPS for constants in Go, as it automatically make them exported (public). Follow the same pattern as with the `deezerAgentName` above.